configs/benchmark/templates/wrk2.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: wrk2-prometheus
+  namespace: {{.Release.Namespace}}
+  labels:
+    jobgroup: wrk2-prometheus
+spec:
+  template:
+    metadata:
+      name: wrk2-prometheus
+      labels:
+        jobgroup: wrk2-prometheus
+        app: wrk2-prometheus
+        custom-affinity: load-generator-node
+    spec:
+      restartPolicy: Never
+      tolerations:
+      - effect: NoSchedule
+        key: load-generator-node
+        operator: Exists
+      volumes:
+      - name: tempfs
+        emptyDir:
+          medium: Memory
+      shareProcessNamespace: true
+      containers:
+      - name: wrk2-prometheus
+        image: quay.io/kinvolk/wrk2-prometheus
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: tempfs
+          mountPath: "/tmpfs"
+        args:
+        - -p
+        {{- if empty .Values.wrk2.serviceMesh }}
+        - "http://pushgateway.{{.Release.Namespace}}:9091/metrics/job/bare-metal/instance/{{.Values.wrk2.app.name}}/run/{{ now | date "20060102150405" }}"
```

Comment:
Can we please use YYYY-MM-DD_HH:MM:SS? This string will end up in a drop-down field in the "wrk2-cockpit" grafana dashboard. It's a lot more readable in the previous format.